RetailCoder.VBE/UI/RubberduckUI.resx

```diff
@@ -381,15 +381,6 @@ Warning: All customized settings will be lost.  Your old file will be saved in '
   <data name="Rubberduck_AboutBuild" xml:space="preserve">
     <value>Version {0}</value>
   </data>
-  <data name="TodoMarkerBug" xml:space="preserve">
```

Comment:
99% sure you don't want to remove that. That came in from a recent PR and isn't in your PR's scope, no?

Reply:
Hmm, nice catch. not sure where that came from.